src/Api/Vault/Controllers/SecurityTaskController.cs

```diff
@@ -87,6 +105,37 @@ public async Task<ListResponseModel<SecurityTasksResponseModel>> BulkCreateTasks
         [FromBody] BulkCreateSecurityTasksRequestModel model)
     {
         var securityTasks = await _createManyTasksCommand.CreateAsync(orgId, model.Tasks);
+
+        var securityTaskCiphers = await _getSecurityTasksNotificationDetailsQuery.GetNotificationDetailsByManyIds(orgId, securityTasks);
+
+        // Get the number of tasks for each user
+        var userTaskCount = securityTaskCiphers.GroupBy(x => x.UserId).Select(x => new
+        UserSecurityTasksCount
+        {
+            UserId = x.Key,
+            Email = x.First().Email,
+            TaskCount = x.Count()
+        }).ToList();
+
+        var organization = await _organizationRepository.GetByIdAsync(orgId);
+        await _mailService.SendBulkSecurityTaskNotificationsAsync(organization.Name, userTaskCount);
+
+        foreach (var userSecurityTaskCipher in securityTaskCiphers)
+        {
+            // Get the associated security task for the cipher
+            var securityTask = securityTasks.First(x => x.CipherId == userSecurityTaskCipher.CipherId);
```

Comment:
❓ Similar to my other comment, could we add the `TaskId` to the `_getSecurityTasksNotificationDetailsQuery.GetNotificationDetailsByManyIds` response data model so we don't have to look it up repeatedly here?